nodeup/pkg/model/kubeproxy.go

```diff
@@ -190,6 +191,11 @@ func (b *KubeProxyBuilder) buildPod() (*v1.Pod, error) {
 		sslCertsHost.MountPath = "/etc/ssl/certs"
 	}
 
+	if dns.IsGossipHostname(b.Cluster.Name) {
+		// Map /etc/hosts from host, so that we see the updates that are made by protokube
+		addHostPathMapping(pod, container, "etchosts", "/etc/hosts")
```

Comment:
Are we adding a comment to `/etc/hosts` that we are maintaining it?  I am not sure if we are adding comments to all of the files we have.  This would be an issue that I would file as a separate enhancement.